_includes/x3d-model.html

```diff
@@ -0,0 +1,10 @@
+{% comment %}
+
+Used to format x3d model
+
+Parameters:
+
+source: The url of the x3d model
+caption: (optional) A caption that is displayed under the image.
+
+{% endcomment %}<span class="screenshot"><x3d><scene><inline url="{{ include.source }}"></inline></scene></x3d> {% if include.caption %}<small>{{ include.caption }}</small>{% endif %}</span>
```

Comment:
This is invalid HTML, right? `x3d` and `scene` aren't HTML tags? This isn't a big problem if that is what this library requires, but overall it seems like a bad practice on their part...